### PR TITLE
fix: convert totalRegistrationTime to snake case

### PR DESCRIPTION
**Pull request by ahtesham-quraish**
### Description

- Convert totalRegistrationTime to snake case

#### JIRA

[VAN-1816](https://2u-internal.atlassian.net/browse/VAN-1816)

#### How Has This Been Tested?

- Tested locally

Here is the backend PR https://github.com/openedx/edx-platform/pull/35333

### CODE REVIEW

File: src/forms/registration-popup/index.jsx [hunk 1]
```diff
@@ -284,7 +284,7 @@ const RegistrationForm = () => {
       ...onboardingComponentContext, ...queryParams, ...payload,
     };
     payload = snakeCaseObject(payload);
-    payload.totalRegistrationTime = totalRegistrationTime;
+    payload.total_registration_time = totalRegistrationTime;
```
Comment (syedsajjadkazmii) [inline src/forms/registration-popup/index.jsx:287]:
```
payload.totalRegistrationTime = totalRegistrationTime;
payload = snakeCaseObject(payload);
```

**Review by syedsajjadkazmii — COMMENTED** [review]
not reviewing it but wanted to point this little improvement

File: src/forms/registration-popup/index.jsx (hunk 1 above)
Comment (abdullahwaheed) [reply src/forms/registration-popup/index.jsx:287]: @ahtesham-quraish please take a look at this

**Review by abdullahwaheed — COMMENTED** [review]
please link backend PR here as well

**Review by abdullahwaheed — APPROVED** [review]

**Pull request merged**